lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -3263,7 +3281,20 @@ swift::ModuleDecl *SwiftASTContext::GetModule(const SourceModule &module,
   auto import_diags = getScopedDiagnosticConsumer();
 
   // Perform the import.
+  std::unique_ptr<lldb_private::Progress> progress = std::make_unique<lldb_private::Progress>("Importing Swift modules");
+  ast->SetPreModuleImportCallback(
+    [&progress](llvm::StringRef module_name, bool is_overlay) {
```

Comment:
@adrian-prantl My bad, the swift standard library module was not a good example. Here however, we could load modules are overlay right ? Do you think that the overlay part is relevant information for the user and print the following for instance:
```
Importing Swift Module: Darwin (overlay).
```
Or do you think it's not necessary to mention that it's an overlay:
```
Importing Swift Module: Darwin.
```

Reply:
Yes, because as your example illustrates, they come in pairs. A Clang module, and it's similarly named Swift overlay.

Reply:
Thank you for clarifying :)